app/lib/feed_manager.rb

```diff
@@ -32,26 +32,33 @@ def key(type, id, subtype = nil)
     "feed:#{type}:#{id}:#{subtype}"
   end
 
-  # Check if the status should not be added to a feed
+  # The filter result of the status to a particular feed
   # @param [Symbol] timeline_type
   # @param [Status] status
   # @param [Account|List] receiver
-  # @return [Boolean]
-  def filter?(timeline_type, status, receiver)
+  # @return [void|Symbol] nil, :filter, or :skip_home
+  def filter(timeline_type, status, receiver)
```

Comment:
The previous public method here was `filter?` - and I think `filter` here is new, and now is essentially internal-only implementation for the `filter?` public method ... should `filter` be private?

Reply:
It should not be private, it's also used by `FeedInsertWorker`.